### DISCUSSION TITLE
Feature Request - web types

### DISCUSSION THREAD

**Discussion by TechAkayy** [themesberg/flowbite-vue#175] [Ideas]
Please add web-types https://github.com/JetBrains/web-types#distribution for flowbite-vue. 

All other top vue-based component libraries like vuetify, quasar, naive-ui, bootstrap-vue, ionic/vue, nuxt all auto-generate & support web-types. Thank you!

**Comment by cogor** [comment]
Thank you for the issue, we'll think about this feature
